src/biogeochem/SatellitePhenologyMod.F90

```diff
@@ -153,53 +145,113 @@ subroutine SatellitePhenology(bounds, num_filter, filter, &
-         ! are less than 0.05, set equal to zero to prevent numerical
-         ! problems associated with very small lai and sai.
-
-         ! snow burial fraction for short vegetation (e.g. grasses, crops) changes with vegetation height
-         ! accounts for a 20% bending factor, as used in Lombardozzi et al. (2018) GRL 45(18), 9889-9897
-
-         ! NOTE: The following snow burial code is duplicated in CNVegStructUpdateMod.
-         ! Changes in one place should be accompanied by similar changes in the other.
-
-         if (patch%itype(p) > noveg .and. patch%itype(p) <= nbrdlf_dcd_brl_shrub ) then
-            ol = min( max(snow_depth(c)-hbot(p), 0._r8), htop(p)-hbot(p))
-            fb = 1._r8 - ol / max(1.e-06_r8, htop(p)-hbot(p))
-         else
-            fb = 1._r8 - (max(min(snow_depth(c),max(0.05,htop(p)*0.8_r8)),0._r8)/(max(0.05,htop(p)*0.8_r8)))
+            tlai_tinterp(p) = timwt(1)*mlai2t(p,1) + timwt(2)*mlai2t(p,2)
          endif
 
-         ! area weight by snow covered fraction
-         if(.not.use_fates_sp)then
+         tsai_tinterp(p) = timwt(1)*msai2t(p,1) + timwt(2)*msai2t(p,2)
+         htop_tinterp(p) = timwt(1)*mhvt2t(p,1) + timwt(2)*mhvt2t(p,2)
+         hbot_tinterp(p) = timwt(1)*mhvb2t(p,1) + timwt(2)*mhvb2t(p,2)
+         
+      end do
+      end associate
+      
+   end subroutine GetSatellitePhenologyInputs
+   
+   !==============================================================================
+   
+   subroutine SetSatellitePhenologyBGCCanopyStructs(bounds, num_filter, filter, &
+    waterdiagnosticbulk_inst, canopystate_inst)
+    !
+    ! !DESCRIPTION:
+    ! Ecosystem dynamics: phenology, vegetation
+    ! Sets the canopystate_inst% data structure for non-FATES runs
+    !
+    ! !USES:
+    use pftconMod               , only : noveg, nbrdlf_dcd_brl_shrub
+    use WaterDiagnosticBulkType , only : waterdiagnosticbulk_type
+    use CanopyStateType         , only : canopystate_type
+    use PatchType               , only : patch
+    use clm_varctl              , only : use_fates_sp
+    
+    !
+    ! !ARGUMENTS:
+    type(bounds_type)              , intent(in)    :: bounds
+    integer                        , intent(in)    :: num_filter                        ! number of column points in patch filter
+    integer                        , intent(in)    :: filter(bounds%endp-bounds%begp+1) ! patch filter points
+    type(waterdiagnosticbulk_type) , intent(in)    :: waterdiagnosticbulk_inst
+    type(canopystate_type)         , intent(inout) :: canopystate_inst
+    !
+    ! !LOCAL VARIABLES:
+    integer  :: fp,p,c                            ! indices
+    real(r8) :: ol                                ! thickness of canopy layer covered by snow (m)
+    real(r8) :: fb                                ! fraction of canopy layer covered by snow
+    
+    if (use_fates_sp) then 
+      write(iulog,*) 'SetSatellitePhenologyBGCCanopyStructs', 'should not be calling this method when use_fates_sp == .true.'
```

Comment:
Remove the subroutine name. The errMsg serves this purpose. See #1452

Reply:
It's actually for any FATES simulation, so maybe should be use_fates? 

And actually maybe this check should just be done at initialization? But, since it's a subroutine call -- there isn't a way to do that. Hmmm.... A single check like this is fine